platform/security/core/security-core-impl/src/main/java/ddf/security/assertion/jwt/impl/SecurityAssertionJwt.java

```diff
@@ -184,9 +184,13 @@ private String getMainPrincipalAsString() throws NoSuchElementException {
       try {
         return (String) attributes.get(claim);
       } catch (NoSuchElementException e) {
-        LOGGER.debug(
-            "Could not find username claim [%s] in jwt claims [%s]",
-            claim, String.join(",", attributes.keySet()), e);
+        if (LOGGER.isDebugEnabled()) {
```

Comment:
❓ Why do we now need to wrap this call in a check for `isDebugEnabled()`?

Reply:
I just decided to wrap this in an `isDebugEnabled()` since one of the arguments is `String.join(",", attributes.keySet())`. This check will prevent that from executing when `debug` is not enabled.

Reply:
Ok, no big deal - I saw on the logging page you linked in your other PR's comments that this sort of wrapping is common when a logging is at debug level and "expensive". I doubt that this is so expensive but it also doesn't hurt anything to have it wrapped. LGTM 👍